### ISSUE TITLE
#4 edit data on details page

### ISSUE THREAD

**Issue by GregOnOff** [GregOnOff/capstone-project#4]
## Value Proposition
​
**As a** :  User
**I want to** edit the data of my items
**so that I**  can correct any mistakes, add links or update the pricing
​
## Description

![Image](https://user-images.githubusercontent.com/113910034/202012928-257549a6-703d-44db-bfa0-a014b950877f.PNG)

​
## Acceptance Criteria
​

- [ ] The details page should contain an edit button
- [ ] The edit button should open a form in which the user can update or change data
- [ ] The form should contain two buttons for saving and exit the form
- [ ] The save button should update the data 
- [ ] The exit button should led back to details page
- [ ] A message should appear when the data is successfully saved 

​
## Tasks
​
- [ ] create a `changeForm` with all properties that can be changed
- [ ] create `backButton` 
- [ ] create a `saveButton`
- [ ] write a function for the `back-Button` that leads back to details page
- [ ] write a function for the `saveButton` that replaces the data in `dummy-data` 
- [ ] add initial styling to the components

## Complexity 

L

**Comment by ahohnsen** [comment]
> The data inside the form is prefilled with the already given data

I would rephrase that sentence:

> The input fields inside the form are prefilled with the already given data
